src/components/Faq/questions.js

```diff
@@ -0,0 +1,29 @@
+        <a href="https://events.decentraland.org/schedule/?id=88d3c09d-1518-4b27-8cca-29099443b2b3&utm_source=landingpage&utm_medium=organic&utm_campaign=DCLMF23" target="_blank">festival event page</a> </p>
+    },
+    {
+        question: 'Where can I find the full festival schedule?',
+        answer: () => <p>The full schedule is above, but you can visit the <a href="https://events.decentraland.org/" target="_blank">Events Page DCLMF23</a> tab to get links to each individual event and jump right in.</p>
+    },
+    {
+        question: 'Do I need a crypto wallet to attend the festival?',
+        answer: () => <p>You don’t need a crypto wallet to attend the festival, or to enter Decentraland for that matter. You can enter as a guest without creating an account. However, if you would like to collect free Wearables as well as further customize your avatar’s look and moves with community-made Wearables and Emotes from the <a href="https://market.decentraland.org/DCLMF23?assetType=item&section=wearables&vendor=decentraland&page=1&sortBy=recently_listed&onlyOnSale=true" target="_blank">Marketplace</a> (and in general save your avatar for return visits to Decentraland) you should sign up for a crypto wallet. Getting a crypto wallet is a simple process and you don’t need to enter any payment details if you just want to save your avatar and claim freebies. Learn more about getting a crypto wallet <a href="https://docs.decentraland.org/player/blockchain-integration/get-a-wallet/" target="_blank">here</a>.</p>
+    },
+    {
+        question: 'How do I get my avatar festival ready with special Wearables and Emotes?',
+        answer: () => <p>In addition to the free basic Wearables and Emotes that everyone can access upon entering Decentraland, you can also get Wearables and Emotes that have been created by the community from the Decentraland Marketplace.<br />In preparation for the Decentraland Music Festival, the Decentraland community created a wide variety of both Wearables and Emotes for the special occasion. You can find them all in a special Decentraland Music Festival tab of the <a href="https://market.decentraland.org/MF23?assetType=item&section=wearables&vendor=decentraland&page=1&sortBy=recently_listed&onlyOnSale=true" target="_blank">Marketplace</a>.<br />To purchase Wearables or Emotes, you’ll need a crypto wallet and MANA, the currency of Decentraland. Learn more about that <a href="https://docs.decentraland.org/player/blockchain-integration/get-a-wallet/" target="_blank">here</a>.</p>
+    },
+    {
+        question: 'How do I keep up to date with festival news, lineup announcements, etc?',
+        answer: () => <p>That’s easy. Just follow <a href="https://twitter.com/decentraland" target="_blank">@Decentraland</a> and #DCLMF23 on Twitter and subscribe to the Decentraland newsletter</p>
```

Comment:
Please, include the `rel="noopener"` attributes in all the anchor elements:
```suggestion
        answer: () => <p>That’s easy. Just follow <a href="https://twitter.com/decentraland" target="_blank" rel="noopener">@Decentraland</a> and #DCLMF23 on Twitter and subscribe to the Decentraland newsletter</p>
```